aten/src/ATen/CMakeLists.txt

```diff
@@ -84,6 +84,9 @@ file(GLOB native_sparse_cuda_cpp "native/sparse/cuda/*.cpp")
 file(GLOB native_quantized_cuda_cu "native/quantized/cuda/*.cu")
 file(GLOB native_quantized_cuda_cpp "native/quantized/cuda/*.cpp")
 
+file(GLOB native_foreach_cu "native/cuda/foreach/*.cu")
+file(GLOB native_foreach_h "native/cuda/foreach/*.h" "native/cuda/foreach/*.cuh")
```

Comment:
Adding a new folder for this feels like pain for no gain (you gonna have to update Buck TARGETs too now...)

Reply:
I'm ok moving it all under native/cuda/ but im curious why we are not separating things like that into separate folders to have a better structure. Eventually we will end up with native/cuda/ having hundred files. 
Is it just because of complexity of MakeFiles?